package.json

```diff
@@ -5,6 +5,7 @@
   "main": "./api.js",
   "author": "",
   "dependencies": {
+    "es6-shim": "^0.27.1",
```

Comment:
Can we rather use core-js here do you think? Since it is already in the dependency chain through Babel.